### PR TITLE
fix #79915 gsm1x.xyz

### LINKED ISSUE #79915: www.gsm1x.xyz
### Issue URL (Anti Adblock Script)
 [https://www.gsm1x.xyz/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.gsm1x.xyz%2F)
### Comment
Username: @jimmysofat6864
> Adblock detected

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.34
Browser: | Chrome
Stealth mode: | disabled
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base, <br/>AdGuard Mobile Ads, <br/>EasyList, <br/>Peter Lowe's Blocklist<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection, <br/>EasyPrivacy, <br/>Fanboy's Enhanced Tracking List<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media, <br/>Fanboy's Social Blocking List<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances, <br/>Fanboy's Annoyances, <br/>Web Annoyances Ultralist, <br/>Adblock Warning Removal List, <br/>I don't care about cookies, <br/>EasyList Cookie List
Other extensions used: | Kaspersky

### PR DESCRIPTION
**Pull request by wout-codes**
#79915

### CODE REVIEW

File: SocialFilter/sections/whitelist.txt [hunk 1]
```diff
@@ -12,6 +12,11 @@ designervn.net#@#.fb-share-button
 !#endif
 ! #End Social locker
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/79915
+gsm1x.xyz#@#.social-icons
```
Comment (zloyden) [inline SocialFilter/sections/whitelist.txt:16]: I agree about this.

File: SocialFilter/sections/whitelist.txt [hunk 2]
```diff
@@ -12,6 +12,11 @@ designervn.net#@#.fb-share-button
...
+gsm1x.xyz#@##fb-root:not(body):not(html)
+gsm1x.xyz#@#.fb-page
+@@||facebook.com/v4.0/plugins/page.php$domain=gsm1x.xyz
```
Comment (zloyden) [inline SocialFilter/sections/whitelist.txt:19]: There is no need to unblock the social widget. Remove leftover.

**Review by zloyden — CHANGES_REQUESTED** [review]

File: SocialFilter/sections/whitelist.txt (hunk 2 above)
Comment (wout-codes) [reply SocialFilter/sections/whitelist.txt:19]: How is that a social widget? Convince me.

Comment (wout-codes) [reply SocialFilter/sections/whitelist.txt:19]: It's a Facebook Fanpage.

Comment (zloyden) [reply SocialFilter/sections/whitelist.txt:19]: > How is that a social widget?

I don't undestand why it's not a social widget? We have a policy to block such elements. In addition, it contains the `Like` and `Share` buttons.

<details><summary>Screenshot:</summary>

image

</details><br/>

Comment (wout-codes) [reply SocialFilter/sections/whitelist.txt:19]: Convinced enough. Thank you.

Comment (zloyden) [reply SocialFilter/sections/whitelist.txt]: You are welcome.

**Review by zloyden — APPROVED** [review]

**Comment by wout-codes** [comment]
No wait.

**Comment by wout-codes** [comment]
I will block the whole thing.

**Comment by wout-codes** [comment]
You're good to go.

**Review by zloyden — APPROVED** [review]

**Comment by zloyden** [comment]
Btw, well done.

**Pull request merged**